tests/Gelf/Test/Transport/RetryTransportWrapperTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+
+namespace Gelf\Test\Transport;
+
+
+use Gelf\Message;
+use Gelf\TestCase;
+use Gelf\Transport\AbstractTransport;
+use Gelf\Transport\RetryTransportWrapper;
+use \PHPUnit_Framework_MockObject_MockObject as MockObject;
+
+class RetryTransportWrapperTest extends TestCase
```

Comment:
Since you're creating a unit test, I think it's a good practice to add a `@covers` annotation to be sure that those tests won't affect the coverage of other classes.

```suggestion
/**
 * @covers RetryTransportWrapper
 */
class RetryTransportWrapperTest extends TestCase
```

See https://phpunit.readthedocs.io/fr/latest/annotations.html#covers for more details.